variables.tf

```diff
@@ -1,49 +1,122 @@
+########################################
+# General vars
+########################################
+variable "description" {
+  default     = ""
+  description = "Description to add to Secret"
+  type        = string
+}
+
+variable "name" {
+  default     = ""
+  description = "Name (omit to use name_prefix)"
+  type        = string
+}
+
+variable "name_prefix" {
+  default     = "terraform"
+  description = "Name Prefix (not used if name specified)"
+  type        = string
+}
+
+variable "pass_version" {
+  default     = 1
+  description = "Password version. Increment this to trigger a new password."
+  type        = number
+}
+
 variable "tags" {
-  default = {}
-  type    = map(string)
+  default     = {}
+  description = "Tags to add to supported resources"
+  type        = map(string)
 }
 
+########################################
+# Secret Notification Rules
+########################################
+
+variable "cloudtrail_log_group" {
+  default     = ""
```

Comment:
This is failing tflint. Maybe get rid of the default or change it to something else?

Reply:
Uggh. It forces the user to set a val if not a default. Will just set it to nonsense I guess